backend/src/main/kotlin/no/uib/echo/plugins/CORS.kt

```diff
@@ -5,10 +5,16 @@ import io.ktor.http.HttpMethod
 import io.ktor.server.application.Application
 import io.ktor.server.application.install
 import io.ktor.server.plugins.cors.routing.CORS
+import no.uib.echo.Environment
 
-fun Application.configureCORS() {
+fun Application.configureCORS(env: Environment) {
     install(CORS) {
-        anyHost()
+        if (env === Environment.PRODUCTION) {
+            allowHost("echo.uib.no", schemes = listOf("https"))
+        } else {
+            allowHost("localhost:3000", schemes = listOf("http"))
```

Comment:
liten kommentar her: måtte legge til `:3000` for at det skulle funke. Det vil ikke være noen grunn for at noen kjører frontend på en anne port en 3000, så burde ikke være noe problem.